second session/assignment/shoppingcart.js

```diff
@@ -0,0 +1,134 @@
+document.addEventListener('DOMContentLoaded', function() {
+    var headerIcon = document.querySelector('.header-icon');
+
+    if (headerIcon) {
+        headerIcon.addEventListener('click', function() {
+            
+            window.location.href = 'shoppingmall.html';
+        });
+    }
+});
+
+document.addEventListener('DOMContentLoaded', function() {
+    var cart__home = document.querySelector('.cart__home');
+
+    if (cart__home) {
+        cart__home.addEventListener('click', function() {
+            
+            window.location.href = 'shoppingmall.html';
+        });
+    }
+});
+
+
+
+
+
+
+// 페이지가 로드될 때 실행되는 함수
+// script.js
+document.addEventListener('DOMContentLoaded', () => {
+    displayCartItems();
+});
+
+function displayCartItems() {
+    const cartItems = JSON.parse(localStorage.getItem('mylist')) || [];
+    const cartTableBody = document.querySelector('#cart_container tbody');
+    cartTableBody.innerHTML = ''; // 기존 목록을 비움
+
+    cartItems.forEach((item, index) => {
+        const row = document.createElement('tr');
+
+        row.innerHTML = `
+            <td><input type="checkbox" class = "checkbox"></td>
+            <td>
+                <div class="product1">
+                    <img src="${item.imgSrc}" alt="상품 이미지" class="product-image">
+                    
+                </div>
+            </td>
+            <td>    
+                <div class="product1">
+                    <h4>${item.name}</h4>
+                </div>
+            </td>
+
+
+
+            <td>
+                <div class="product-price">${formatPrice(item.price)}원</div>
+            </td>
+            <td>
+                <div class="product-category">${item.category}</div>
+            </td>
+            <td>
+                <div class="delete-btn" type ="button" onclick="removeItemFromCart(${index})">삭제</div>
+            </td>
+        `;
+
+        cartTableBody.appendChild(row);
+    });
+}
+
+function removeItemFromCart(index) {
+    let cartItems = JSON.parse(localStorage.getItem('mylist')) || [];
+    cartItems.splice(index, 1); // 특정 인덱스의 품목을 삭제
+    localStorage.setItem('mylist', JSON.stringify(cartItems)); // 업데이트된 목록을 다시 저장
+    displayCartItems(); // 목록을 다시 표시
+}
+
+function formatPrice(price) {
+    return price.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
```

Comment:
p5: 정규 표현식 사용하셨군요!  간단하게 toLocaleString()을 사용하는 방법도 추천해 드리고 싶어요~ 아래 링크로 소개해드릴게요! 
https://shape-coding.tistory.com/72

```suggestion
function formatPrice(price) {
    return Number(price).toLocaleString() // price가 이미 Number 타입이라면 Number는 생략해도됩니다
}
```

Reply:
아 toLocaleString()이라는 것도 있었군여 감사합니당

Reply:
p5: 저두 공부하고 갑니다 22~